ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -741,7 +749,62 @@
           - email:
               notify-every-unstable-build: true
               recipients: projectantrea-dev@googlegroups.com
-          wrappers: []
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: WORKFORCE_POOL # Jenkins secret that stores client id
+                variable: WORKFORCE_POOL
+      - 'cloud-{name}-{test_name}':
+          test_name: credential-for-period
+          node: antrea-credential
+          description: 'This is the cloud credential job for antrea gke tests.'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              set +x
```

Comment:
Is `set +x` for avoiding exposing the credentials from the console output? I think this is necessary.

Reply:
Yes, we don't want it expose in jenkins log.

Reply:
Maybe also add a comment here. This is to avoid other developers removing this set+x by accident.